### PR TITLE
git-terminal: fix typos and warnings

### PR DESCRIPTION
**Pull request by kbdharun**
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

## Changes

This PR fixes typos and broken renderings of warnings.

### CODE REVIEW

File: contributing-guides/git-terminal.md [hunk 1]
```diff
@@ -14,8 +14,8 @@ The overall process should look somewhat like this:
 3. Create a feature branch, e.g. named after the command you plan to edit:
   `git checkout -b {{branch_name}}`
 
-   > [!WARNING]
-   > It is bad practice to submit a PR from the `main` branch of your forked repository. Please create pull requests from a well named feature branch.
+> [!WARNING]
+  > It is bad practice to submit a PR from the `main` branch of your forked repository. Please create pull requests from a well-named feature branch.
```
Comment (acuteenvy) [inline contributing-guides/git-terminal.md:18]:
```suggestion
> [!WARNING]
> It is bad practice to submit a PR from the `main` branch of your forked repository. Please create pull requests from a well-named feature branch.
```

File: contributing-guides/git-terminal.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ Typically a pull request will include changes in a single file **unless the pull
 
 # Updating your fork
 
-Forks of GitHub repositories aren't updated automatically. To keep your fork up-to-date with the latest changes and avoid merge conflicts, you should update it regularly.
+Forks of GitHub repositories aren't updated automatically. You should update your fork regularly to keep your fork up-to-date with the latest changes and avoid merge conflicts.
```
Comment (acuteenvy) [inline contributing-guides/git-terminal.md:43]:
```suggestion
Forks of GitHub repositories aren't updated automatically. You should update your fork regularly to keep it up-to-date with the latest changes and avoid merge conflicts.
```

**Review by acuteenvy — APPROVED** [review]
> broken renderings of warnings

This was changed a month ago - hopefully they bring it back, because now the indentation looks out of place.
https://github.com/orgs/community/discussions/16925
> #### Update - 14 November 2023
> Prevent alerts from being nested within other elements.

**Comment by kbdharun** [comment]
> This was changed a month ago - hopefully they bring it back, because now the indentation looks out of place.
https://github.com/orgs/community/discussions/16925

Yeah, I heard about it too. Will apply your suggestions.

**Pull request merged**